Programmers/230901_푸드_파이트_대회/kwaksj329.js

```diff
@@ -0,0 +1,12 @@
+function solution(food) {
+  let original = [];
+  for (let i = 1; i < food.length; i++) {
+    for (let j = 0; j < Math.floor(food[i] / 2); j++) {
+      original.push(i);
+    }
+  }
+  let reverse = [...original].reverse();
```

Comment:
```suggestion
let reversed = [...original].reverse();
```
`reverse`라는 이름을 가진 함수가 있기 때문에 혼란 방지를 위해 변수는 `reversed`와 같이 다른 이름을 쓰면 더 좋을 것 같아요~ 
`reverse` 함수가 in place니까 spread 해줘야 하는 부분은 잘 캐치하셨네요 :)